docs/src/main/sphinx/security/password-file.rst

```diff
@@ -14,8 +14,15 @@ and clients to use TLS and authenticate with a username and password.
 Password authenticator configuration
 ------------------------------------
 
-Enable password file authentication by creating an
-``etc/password-authenticator.properties`` file on the coordinator:
+Enable password file authentication by setting the :doc:`password authentication
+type <authentication-types>` in ``config.properties``:
```

Comment:
`etc/config.properties` to be consistent, since we're using `etc/` below

Reply:
done